src/webpack/webpackTools.ts

```diff
@@ -66,10 +106,18 @@ export function createConfig(outPath: string, entryPoint: string | webpack.Entry
         }
         config.output!.libraryTarget = 'umd';
     }
+
     if (plugins && plugins.length) {
         config.plugins!.push.apply(config.plugins, plugins);
     }
 
+    if (extractCSS) {
+        const name = options!.extractStyles!;
+        config.plugins!.push(new MiniCssExtractPlugin({
+            filename: (typeof name === 'string') ? `${name}.css` : '[name].css'
```

Comment:
Is "[name]" something MiniCssExtractPlugin knows how to parse?

Reply:
Yeah the name gets passed to it from webpack just like the `output`: `[name]-bundle.js`